recipes/r-boot/build.sh

```diff
@@ -1,13 +0,0 @@
-#!/bin/bash
-
-# R refuses to build packages that mark themselves as Priority: Recommended
-mv DESCRIPTION DESCRIPTION.old
-grep -v '^Priority: ' DESCRIPTION.old > DESCRIPTION
```

Comment:
@johanneskoester this is the `default` we get from conda-skeleton. I also remove it, but did you ever find a case where that was needed? (Just curious.)